src/azure-cli/azure/cli/command_modules/iot/_validators.py

```diff
@@ -75,3 +76,20 @@ def validate_c2d_ttl(ns):
             ns.c2d_ttl not in range(1, 49, 1)):
         raise ArgumentError(None, 'Please specify the default time from 1 to 48 hours to live for cloud-to-device'
                                   ' messages in the device queue.')
+
+
+def validate_private_endpoint_connection_id(namespace):
```

Comment:
do we still need this function?

Reply:
Nope, must have missed that in the network code refactor. Will remove.